### DISCUSSION TITLE
how to start job at fix time in Quartz 3.3.2

### DISCUSSION THREAD

**Discussion by surajit-te** [quartznet/quartznet#1248] [Q&A]
how to start job at fix time of a day

**Comment by lahma** [comment]
Please read the [tutorial](https://www.quartz-scheduler.net/documentation/quartz-3.x/tutorial/). You can for example use SimpleTrigger (set specific start time and 24h interval,  or Crontrigger with pattern that suites you.

**Reply by surajit-te** [reply]
But i start the first trigger at fix time with 1h interval its work
